packages/react-native-editor/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gutenberg-mobile",
+  "name": "@wordpress/react-native-editor",
   "version": "1.16.1",
   "private": true,
```

Comment:
`devDepenendcies` should be removed from packages. We hoist them and keep in the root folder as all tools are managed globally, see https://github.com/WordPress/gutenberg/tree/master/packages#development-dependencies:

> In contrast to production dependencies, development dependencies shouldn't be stored in individual WordPress packages. Instead they should be installed in the project's `package.json` file using the usual npm install command. In effect, all development tools are configured to work with every package at the same time to ensure they share the same characteristics and integrate correctly with each other.